scripts/assets/file_content_validation_checks.textproto

```diff
@@ -271,6 +271,7 @@ file_content_checks {
   exempted_file_name: "app/src/main/java/org/oppia/android/app/player/audio/LanguageDialogFragment.kt"
   exempted_file_name: "app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt"
   exempted_file_name: "app/src/sharedTest/java/org/oppia/android/app/home/HomeActivityTest.kt"
+  exempted_file_name: "app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt"
```

Comment:
@BenHenning, not sure there is a way to use OppiaLocale in forcing default locale in tests, so I exempted the test file from this check.

Reply:
Should be a fine exemption to use. I generally don't mind tests using ``Locale`` (and in fact, maybe we should just default allow all tests to avoid needing to update this in the future, but I don't feel strongly one way or the other).

Reply:
Going ahead and resolving this since it seems addressed.